src/background.js

```diff
@@ -65,20 +76,30 @@ function handleWake() {
   const now = Date.now();
   log('woke at', now);
   return browser.storage.local.get().then(items => {
-    const due = Object.values(items).filter(item => item.time <= now);
+    const due = Object.entries(items).filter(entry => entry[1].time <= now);
     log('tabs due to wake', due.length);
-    return Promise.all(due.map(item =>
-      browser.tabs.create({
-        active: false,
-        url: item.url,
-        windowId: item.windowId
-      }).then(tab => browser.notifications.create(`${item.windowId}:${tab.id}`, {
-        'type': 'basic',
-        'iconUrl': browser.extension.getURL('link.png'),
-        'title': item.title,
-        'message': item.url
-      })).then(() => browser.storage.local.remove(idForItem(item)))
-    ));
+    return browser.windows.getAll({
+      windowTypes: ['normal']
+    }).then(windows => {
+      let windowIds = windows.map(window => window.id);
+      return Promise.all(due.map(([id, item]) => {
```

Comment:
ESLint is whining because `id` is apparently unused, but since we're destructuring an array here, not sure if we should do an inline `// eslint-disable-lint no-unused-vars` or if we can do something classier:

```
/Users/pdehaan/dev/github/mozilla/SnoozeTabs/src/background.js
  85:36  warning  'id' is defined but never used  no-unused-vars
```